Misc/NEWS.d/next/Windows/2024-10-31-09-46-53.gh-issue-125729.KdKVLa.rst

```diff
@@ -0,0 +1 @@
+Mention the turtle module in Tcl/Tk installation option in Windows installer. Exclude turtle.py from stdlib if this option is not set.
```

Comment:
```suggestion
Mention the :mod:`turle` module in the Tcl/Tk installation option of the
Windows installer. If the option is not set, :file:`Lib/turtle.py` will be
excluded from the standard library and will not present on the host machine.
```

I *think* this could do the job but I'm not entirely sure that the links will work properly so we'll need to wait for the docs CI to build.

Reply:
No need to refer to the file in my opinion, but we do need to spell ``:mod:`turtle` `` right.

I'd probably phrase this as ``Makes the presence of the :mod:`turtle` module dependent on the Tcl/Tk installer option. Previously, the module was always installed but would be unusable without Tcl/Tk.``

Reply:
Oups, sorry for the typo !